introduction/intro.md

```diff
@@ -13,166 +13,30 @@ redirect_from:
-open-source software (FOSS). This means that everyone is free to use, copy,
-and change the software in any way. It also means that the source code is
-openly available so others can contribute to and audit it.
-
-Why is OS security important?
------------------------------
-
-Most people use an operating system like Windows or OS X on their desktop
-and laptop computers. These OSes are popular because they tend to be easy
-to use and usually come pre-installed on the computers people buy. However,
-they present problems when it comes to security. For example, you might
-open an innocent-looking email attachment or website, not realizing that
-you're actually allowing malware (malicious software) to run on your
-computer. Depending on what kind of malware it is, it might do anything
-from showing you unwanted advertisements to logging your keystrokes to
-taking over your entire computer. This could jeopardize all the information
-stored on or accessed by this computer, such as health records, confidential
-communications, or thoughts written in a private journal. Malware can also
-interfere with the activities you perform with your computer. For example,
-if you use your computer to conduct financial transactions, the malware
-might allow its creator to make fraudulent transactions in your name.
-
-Aren't antivirus programs and firewalls enough?
------------------------------------------------
-
-Unfortunately, conventional security approaches like antivirus programs
-and (software and/or hardware) firewalls are no longer enough to keep out
-sophisticated attackers. For example, nowadays it's common for malware
-creators to check to see if their malware is recognized by any signature-based
-antivirus programs. If it's recognized, they scramble their code until it's
-no longer recognizable by the antivirus programs, then send it out. The
-best of these programs will subsequently get updated once the antivirus
-programmers discover the new threat, but this usually occurs at least a
-few days after the new attacks start to appear in the wild. By then, it's
-too late for those who have already been compromised. More advanced antivirus
-software may perform better in this regard, but it's still limited to a
-detection-based approach. New zero-day vulnerabilities are constantly being
-discovered in the common software we all use, such as our web browsers, and no
-antivirus program or firewall can prevent all of these vulnerabilities from
-being exploited.
+Qubes OS is a free and open-source security-oriented operating system meant for single-user desktop computing. 
 
+Qubes OS allows you to compartmentalize various parts of your digital life into isolated domains. If one of those compartments get compromised by a malicious email attachment carrying a ransomware, other parts of your system will remain unaffected : this is one of the key benefits of the [*security by compartmentalization*](https://www.qubes-os.org/faq/) approach taken by Qubes OS.
```

Comment:
"If one of those compartments" -> "If one of those domains"

Reply:
Done